tests/features/call_proposals.feature

```diff
@@ -0,0 +1,112 @@
+  In order to have Calls for proposals on the site
+  As an editor
+  I need to be able to create, edit and see the call for proposals items
+
+  @javascript
+  Scenario: Creation of a Call for proposals content through the UI.
+    Given I am logged in as a user with the "create oe_call_proposals content, access content, edit own oe_call_proposals content, view published skos concept entities, manage corporate content entities" permission
+
+    And the following document:
+      | name          | file       |
+      | My Document 1 | sample.pdf |
+
+    And the following images:
+      | name          | file           | alt                            |
+      | Contact image | example_1.jpeg | Contact image alternative text |
+
+    When I visit "the Call for proposals creation page"
+    Then I should have the following options for the "Deadline model" select:
+      | - Select a value - |
+      | Single-stage       |
+      | Two-stage          |
+      | Multiple cut-off   |
+      | Permanent          |
+    And I fill in "Title" with "My Call for proposals 1"
+    And I set "Publication date" to the date "24-10-2020"
+
+    # Test that no Deadline Date field is visible when the Permanent model is selected
+    And I select "Permanent" from "Deadline model"
+    And I should not see "Deadline date"
+
+    And I select "Two-stage" from "Deadline model"
+    And I set "Deadline date" to the date "31-12-2020 23:45" using format "d-m-Y H:i"
+    And I fill in "Content owner" with "Committee on Agriculture and Rural Development (http://publications.europa.eu/resource/authority/corporate-body/EP_AGRI)"
+
+    And I press "Save"
+
+    Then I should see "Call for proposals My Call for proposals 1 has been created."
+    And I should see "My call for proposals 1"
+    And I should see "10/24/2020"
+    And I should see "Two-stage"
+    And I should see "12/31/2020 - 23:45"
+
+    When I click "Edit"
+    And I fill in "Body text" with "My Call for proposals 1 body"
+    And I fill in "Reference" with "My Call for proposals 1 reference"
+    And I fill in "URL" with "http://example.com/1" in the "Publication in the official journal" region
+    And I fill in "Link text" with "Official Journal publication 1" in the "Publication in the official journal" region
+    And I set "Opening date" to the date "25-10-2020"
+    And I fill in "Grants awarded link" with "http://example.com/2"
+    And I fill in "Funding programme" with "Anti Fraud Information System (AFIS) (http://publications.europa.eu/resource/authority/eu-programme/AFIS2020)"
+    And I fill in "Responsible department" with "Audit Board of the European Communities (http://publications.europa.eu/resource/authority/corporate-body/ABEC)"
+    And I fill in "Use existing media" with "My Document 1" in the "Documents" region
+
+    # Call for proposals contact field group.
+    When I press "Add new contact" in the "Call for proposals contact" region
+    And I wait 10 seconds
```

Comment:
`And I wait for AJAX to finish` -  there is a setting extensions.ajax_timeout in behat.yml that defines how long need to wait.